docs/conf.py

```diff
@@ -0,0 +1,72 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
Can you please copy the IBM copyright notice from one of our other `.py` files into the top of this file?